packages/mobile/src/components/now-playing-drawer/PlayButton.tsx

```diff
@@ -1,19 +1,24 @@
-import { useCallback } from 'react'
+import { useCallback, useEffect, useState } from 'react'
 
 import { playerActions, playerSelectors } from '@audius/common/store'
 import { useDispatch, useSelector } from 'react-redux'
 
 import IconPause from 'app/assets/animations/iconPause.json'
 import IconPlay from 'app/assets/animations/iconPlay.json'
+import IconLoadingSpinner from 'app/assets/animations/iconPlayLoadingSpinner.json'
 import type { AnimatedButtonProps } from 'app/components/core'
 import { AnimatedButton } from 'app/components/core'
 import { makeAnimations } from 'app/styles'
 import { colorize } from 'app/utils/colorizeLottie'
 import { Theme } from 'app/utils/theme'
+
 const { pause, play } = playerActions
-const { getPlaying } = playerSelectors
+const { getPlaying, getBuffering } = playerSelectors
+
+// Minimum of how long spent buffering for until we show the spinner
+const MINIMUM_BUFFER_TIME_TO_SHOW = 500 // ms
```

Comment:
Need to verify how we feel about this number

Reply:
https://github.com/AudiusProject/audius-protocol/blob/3af1635df8dddcce5a613b08fd49bb1bdf4036bb/packages/embed/src/audio/AudioPlayer.js#L9 

looks like 1s on web, let's match that for now

Reply:
jk this is the embed player, but web is also 1s